less16 commit1

renvema opened the pull request:
Have a nice day)

Commit:
less16 commit1

Sofasmile commented on src/task/sixteenth/Assert.java:
```diff
@@ -0,0 +1,158 @@
+package task.sixteenth;
+
+import java.util.*;
```
auto import? :)

Sofasmile commented on src/task/sixteenth/Assert.java:
```diff
@@ -0,0 +1,158 @@
+            this.content = content;
+        }
+
+        public final String getFrom() {
```
do we need final here?

Sofasmile commented on src/task/sixteenth/Assert.java:
```diff
@@ -0,0 +1,158 @@
+        };
+
+        @Override
+        public void accept(IMessage<T> message) {
```
try to simplify this method

Sofasmile commented on src/task/sixteenth/Assert.java:
```diff
@@ -0,0 +1,158 @@
+    }
+
+    public static class MailService<T> implements Consumer<IMessage<T>> {
+        Map<String, List<T>> notification = new HashMap() {
```
add access modifier

Commit:
less16 commit2
-change auto import;
-remove "final";
-simplify method;
-add access modifier to Map;

Sofasmile commented on src/task/sixteenth/Assert.java:
```diff
@@ -0,0 +1,160 @@
+    }
+
+    public static class MailService<T> implements Consumer<IMessage<T>> {
+        public Map<String, List<T>> notification = new HashMap() {
```
maybe better use private here?
